vmm/sandbox/src/args.rs

```diff
@@ -0,0 +1,36 @@
+use clap::Parser;
+
+pub mod built_info {
+    // The file has been placed there by the build script.
+    include!(concat!(env!("OUT_DIR"), "/built.rs"));
+}
+
+#[derive(Parser, Debug)]
+#[command(author, about, long_about = None)]
+pub struct Args {
+    /// Version info
+    #[arg(short, long)]
+    pub version: bool,
+
+    /// Config file path, only for cloud hypervisor and stratovirt
+    #[arg(short, long, value_name = "FILE")]
+    pub config: Option<String>,
+
+    /// kuasar working directory
+    #[arg(short, long, value_name = "DIR")]
+    pub dir: Option<String>,
+
+    /// listen, should be a unix domain socket
+    #[arg(short, long, value_name = "FILE")]
+    pub listen: Option<String>,
+}
+
+pub fn print_version_info() {
```

Comment:
This function seems not relevant with the args, maybe the mod built_info mod and this function should be in a seperate file?